### ISSUE TITLE
list size is incorrectly calculated

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#836]
**Reported by simekk on 2010-08-19 20:17**
Using leatest snapshot of NVDA and Firefox 3.6.8

This code:

```
<ul class="nav">
  <li><a href="index.php">Overview</a></li>
  <li><a href="tabpanel1.php">Example 1</a></li>
  <li><a href="tabpanel2.php">Example 2</a></li>
</ul>
```

produces: 
list with 5 items 
visited link Overview  
visited link Example 1  
visited link Example 2

code is from: 
http://test.cita.illinois.edu/aria/tabpanel/tabpanel1.php

Perhaps the blanks are calculated as well but it makes little sense to do so.

**Comment by dkager** [comment]
The actual code that this issue refers to has been lost, maybe in the transition to GitHub? I'd say close unless the code can be re-added.

**Comment by jcsteh** [comment]
The code was still there, but it was being interpreted as real HTML. I've edited the comment to fix this.

Nevertheless, this is the same issue as #2151, which was fixed in ca1d1511. Closinga s duplicate.
